wallet/src/main/scala/org/bitcoins/wallet/models/SpendingInfoTable.scala

```diff
@@ -80,7 +89,39 @@ case class LegacySpendingInfo(
     copy(blockHash = Some(blockHash))
 }
 
-// TODO add case for nested segwit
+/**
+  * DB representation of a nested segwit V0
+  * SegWit UTXO
+  */
+case class NestedSegwitV0SpendingInfo(
+    outPoint: TransactionOutPoint,
+    output: TransactionOutput,
+    privKeyPath: NestedSegWitHDPath,
+    redeemScript: ScriptPubKey,
+    scriptWitness: ScriptWitness,
+    txid: DoubleSha256DigestBE,
+    state: TxoState,
+    id: Option[Long] = None,
```

Comment:
nit: If you move `id` down one then you won't have to mention it when using `None`